code/game/vehicles/spacepods/spacepod.dm

```diff
@@ -785,6 +835,12 @@
 					moveship = 0
 		if(moveship)
 			Move(get_step(src, direction), direction)
+			if(equipment_system.cargo_system)
+				for (var/turf/T in locs)
+					for (var/obj/item/I in T.contents)
+						if(I.loc == T)
```

Comment:
what is this
if something is in turf.contents then it's loc will always be turf